packages/contracts/contracts/token/Managed.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.4.24;
+import "../zeppelin-solidity/ownership/Ownable.sol";
+
+contract Managed is Ownable {
+    mapping (address => bool) public managers;
+    
+    modifier onlyManager () {
+        require(isManager(), "Only managers may perform this action");
+        _;
+    }
+
+    modifier onlyManagerOrOwner () {
+        require(
+            checkManagerStatus(msg.sender) || msg.sender == owner,
+            "Only managers or owners may perform this action"
+        );
+        _;
+    }
+
+    function checkManagerStatus (address managerAddress) public view returns (bool) {
+        return managers[managerAddress];
+    }
+
+    function isManager () public view returns (bool) {
+        return checkManagerStatus(msg.sender);
+    }
+
+    function addManager (address managerAddress) public onlyManagerOrOwner {
```

Comment:
I still think the `addManager` and `removeManager` functions are kind of dangerous. if one manager is compromised the whole thing is.